electron/constants.ts

```diff
@@ -107,11 +107,26 @@ function fixAsarPath(origin: string): string {
   return origin
 }
 
-function getSteamCompatFolder() {
-  if (existsSync(`${userHome}/.var/app/com.valvesoftware.Steam/.steam/steam`)) {
-    return `${userHome}/.var/app/com.valvesoftware.Steam/.steam/steam`
+export function getSteamCompatFolder() {
+  // Paths are from https://savelocation.net/steam-game-folder
+  if (isWindows) {
+    return join('C:', 'Program Files (x86)', 'Steam')
```

Comment:
maybe you can try running the `where steam` command in the system instead of hardcoding this path? I don't have windows but I found it's kinda the equivalent of `which steam` in linux. If a user changes their installation folder this will break

Reply:
oh nevermind, I see this looks for the compat folder, not the steam binary folder